src/main/java/org/mule/extension/db/internal/domain/connection/derby/DerbyConnectionParameters.java

```diff
@@ -36,7 +36,7 @@
   private String database;
 
   /**
-   * Specifies the type of SubsubProtocol to be used by Derby. The available options are:
+   * If your application uses an embedded database, the subsubprotocol parameter specifies where Derby should look for the database. The available options are:
```

Comment:
We should try to avoid lines longer than 120 characters. Include a new line before the word "database"

Reply:
I just took out the extra text. Pro Derby users already know it's for apps that use embedded databases.